tests/Tests.Core/ManagedElasticsearch/Clusters/ClientTestClusterBase.cs

```diff
@@ -23,6 +24,15 @@ public abstract class ClientTestClusterBase : XunitClusterBase<ClientTestCluster
 		public IElasticClient Client => this.GetOrAddClient(s => ConnectionSettings(s.ApplyDomainSettings()));
 
 		protected virtual ConnectionSettings ConnectionSettings(ConnectionSettings s) => s;
+
+		protected sealed override void SeedCluster()
+		{
+			Client.Cluster.Health(new ClusterHealthRequest { WaitForStatus = WaitForStatus.Green });
```

Comment:
should this check the response?

Reply:
I'm okay to fare blind here, let's see how it behaves first before deciding.